src/styles/BookList.module.css

```diff
@@ -0,0 +1,180 @@
+/* For BookList.jsx */
+.booklist {
+  background-color: #fafafa;
+  display: flex;
+  flex-direction: column;
+  margin: 2% 9%;
+}
+
+/* For Book.jsx */
+.bookContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  border-radius: 4px;
+  border: 1px solid #e8e8e8;
+  margin: 2%;
+  background-color: #fff;
+  padding: 1%;
+}
+
+.bookInnerContainer {
+  display: flex;
+  flex-direction: column;
+  margin: 2% 0;
+}
+
+.bookButtons {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.divider {
+  border: 1px solid #e0e0e0;
+  margin: 0 5%;
+}
```

Comment:
- [ ] Kindly make sure your Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/) as the requirements say, you should follow (the layout, colors, padding, margin...). check here the difference


Your design | The template 
-----|-----
<img width="1357" alt="image" src="https://github.com/cancelei/bookstore/assets/65068771/311870ef-cea4-489d-a37e-26d6b0aaa486"> | <img width="1073" alt="image" src="https://github.com/Eccb7/book-store/assets/65068771/76904bb6-aeaf-4ece-8367-75464f2fb3d6">
<img width="1357" alt="image" src="https://github.com/cancelei/bookstore/assets/65068771/c1a8c215-a2f4-4085-a079-cd73308286b4"> | <img width="1053" alt="image" src="https://github.com/Eccb7/book-store/assets/65068771/1ea0b5b7-cbc9-4ffc-918a-0d8857b520c2">